Cargo.toml

```diff
@@ -14,6 +14,14 @@ edition = "2018"
 [features]
 liquid = [ "elements" ]
 electrum-discovery = [ "electrum-client"]
+tracing-enabled = [
```

Comment:
let's call this "tracing"

Reply:
That makes feature name a cyclic dependency with tracing crate. Therefore feature name will be:
```
otlp-tracing = [
    "tracing",
```

It could be workaround by specifying "dep:"
```
tracing = [
    "dep:tracing",
```
but that syntax  on the other hand doesn't work with "tracing/max_level_trace", so this would fail:
```
tracing = [
    "dep:tracing",
    "dep:tracing/max_level_trace",
```